config/rbac/vg_manager_cluster_role_binding.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Nit. Please rename the file to vg_manager_clusterrole_binding.yaml to match the other files.

Reply:
done